packages/dialtone-vue3/components/input/input.vue

```diff
@@ -141,7 +141,7 @@ export default {
      * Name property of the input element
      */
     name: {
-      type: String,
+      type: jwfkahjfklewjfaw,
```

Comment:
What happened here? lol

![giphy](https://media.giphy.com/media/lkdH8FmImcGoylv3t3/giphy.gif)

Reply:
haha oh wow, was testing something with diffs and forgot to revert that